slybot/slybot/plugins/scrapely_annotations/annotations.py

```diff
@@ -211,8 +214,16 @@ def _request_to_follow_from_region(self, htmlregion):
             if request is not None:
                 yield request
 
-    def handle_rss(self, response, seen):
-        for link in self.rss_link_extractor.links_to_follow(response):
```

Comment:
Is `self.rss_link_extractor` still used anywhere? If it isn't remove it from `setup_bot()` as well